client/src/components/Admin/Orders/AddressDetails.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+
+const AddressDetails = ({ orderDetails }) => {
```

Comment:
can we reuse the address component that Gangotri created for the store front?
if yes then these lines can be avoided or simplified, what do you think?

Reply:
I think that Address component is different. Here in this AddressDetails component I have a table and showing the customer address details after clicking on any order